src/shell/views/Shell/Shell.less

```diff
@@ -24,18 +24,38 @@
   height: 100vh;
   overflow: hidden;
   display: grid;
-  grid-template-columns: 60px 1fr;
+  grid-template-columns: 160px 1fr;
 
-  scrollbar-color: @zesty-grey @zesty-light-grey;
-  scrollbar-width: thin;
+  //CLOSED MENU
+  &.NavClosed {
+    grid-template-columns: 60px 1fr;
 
-  @media (min-width: 2000px) {
-    & {
-      grid-template-columns: 170px 1fr;
-      font-size: 14px;
+    a[href="/content"],
+    a[href="/media"],
+    a[href="/schema"],
+    a[href="/code"],
+    a[href="/leads"],
+    a[href="/analytics"],
+    a[href="/seo"],
+    a[href="/audit-trail"],
+    a[href="/settings"],
+    [title="Help"],
+    [title="Zesty.io"] {
+      span {
+        display: none;
+      }
+    }
+    [data-icon="eye"] + span {
+      display: none;
+    }
+    [data-icon="hashtag"] {
+      display: none;
     }
   }
 
+  scrollbar-color: @zesty-grey @zesty-light-grey;
```

Comment:
these are defined at the global app and cascade. do these values differ?

Reply:
@shrunyan Im not 100% following the action on this one as well. Is this referring to scrollbar values?

Reply:
Yes scrollbar values. What's the reason for defining these? These styles should already exist at the shell and cascade to this area.

Reply:
Yes, these existed already here in shell level, I relocated them back to there proper place after looking at previous master https://github.com/zesty-io/manager-ui/blob/f611e263b994f145c3b12fde517af6e7ae0353e8/src/shell/views/Shell/Shell.less#L29-L30